mne/preprocessing/ica.py

```diff
@@ -222,14 +222,18 @@ class ICA(ContainsMixin):
         Channel names resulting from initial picking.
     n_components_ : int
         If fit, the actual number of PCA components used for ICA decomposition.
+    explained_variance_ratio_ : array, shape (n_components_ | None
+        If fit, the relative variance explained by each ICA component. If not
+        fit, ``None``.
     pre_whitener_ : ndarray, shape (n_channels, 1) or (n_channels, n_channels)
         If fit, array used to pre-whiten the data prior to PCA.
     pca_components_ : ndarray, shape ``(n_channels, n_channels)``
         If fit, the PCA components.
     pca_mean_ : ndarray, shape (n_channels,)
         If fit, the mean vector used to center the data before doing the PCA.
     pca_explained_variance_ : ndarray, shape ``(n_channels,)``
-        If fit, the variance explained by each PCA component.
+        If fit, the variance explained by each PCA component. Note that the
+        variances are not unitized, i.e., don't sum to 1.
     mixing_matrix_ : ndarray, shape ``(n_components_, n_components_)``
```

Comment:
the lines above and below the changed line here have the shape in double-backticks (they shouldn't). Do you mind fixing that here?

Reply:
Done!